client/src/components/PredictionPanel.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class='container'>
+    <map-geocoder-wrapper v-if='showSearch' v-model='expandSearch' />
```

Comment:
I'd rather remove the search from this component. We already have a search component parallel to this one and removing it from here would tighten its scope to just displaying predictions/scorecard(s).

Reply:
Yeah this is the search bar that needs to be moved to the landing page. If you want me to take it out of here now that you started the landing page, just let me know. I can also leave it in if this PR is going in first for you to copy / paste over?

Reply:
Sounds good. I'll move it around as necessary in my PR so it's fine as-is here.